website/docs/docs/dbt-versions/core-versions.md

```diff
@@ -20,9 +27,11 @@ All dbt Core versions released prior to 1.0 and their version-specific documenta
 
 ## EOL version support 
 
-All dbt Core versions with an end-of-life (EOL) support level will no longer receive bug fixes. To continue receiving bug fixes, dbt Labs recommends upgrading to a newer version. 
+All dbt Core versions with an end-of-life (EOL) support level will no longer receive bug fixes. We recommend upgrading to a newer version in [dbt Cloud](/docs/dbt-versions/upgrade-core-in-cloud) or [dbt Core](/docs/core/installation#upgrading-dbt-core) to continue receiving bug fix support. 
```

Comment:
> All dbt Core versions with an end-of-life (EOL) support level will no longer receive bug fixes.

We can make this stronger: They'll no longer receive any fixes in new patch releases.